shell/common/variable_refresh_rate_display_unittests.cc

```diff
@@ -0,0 +1,29 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "variable_refresh_rate_display.h"
+#include "vsync_waiters_test.h"
+
+#include "gtest/gtest.h"
+
+namespace flutter {
+namespace testing {
+
+TEST(VariableRefreshRateDisplayTest, ReportCorrectInitialRefreshRate) {
+  auto refresh_rate_reporter = std::make_shared<TestRefreshRateReporter>(60);
```

Comment:
nit: no need for make shared in these tests anymore.

Reply:
done